tests/test_client.py

```diff
@@ -1,11 +1,13 @@
 import json
+
 import pytest
+from swagger_client.rest import ApiException
 
 from indexd.index.blueprint import ACCEPTABLE_HASHES
 
 
-def test_index_list(client):
-    assert client.get('/index/').status_code == 200
+def test_index_list(swg_client):
+    swg_client.list_entries()
```

Comment:
Is this just looking for an error? Doesn't look like a test.

Reply:
this is the same expectation as the previous assert, just that when the status_code is 200, swagger client doesn't raise exception